quickstart/deploy-mayastor.md

```diff
@@ -6,317 +6,99 @@
-```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor/v1.0.2/deploy/etcd/statefulset.yaml 
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor/v1.0.2/deploy/etcd/svc.yaml
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor/v1.0.2/deploy/etcd/svc-headless.yaml
-```
-{% endtab %}
-{% endtabs %}
-
-Verify that the deployment of etcd to the cluster was successful. Within the mayastor namespace there should be 3 replicas with a name of the form "mayastor-etcd-" and with a reported status of "Running".
-
-{% tabs %}
-{% tab title="Command" %}
-```text
-kubectl -n mayastor get pods --selector=app.kubernetes.io/name=etcd
-```
-{% endtab %}
-
-{% tab title="Example Output" %}
-```text
-NAME              READY   STATUS    RESTARTS   AGE
-mayastor-etcd-0   1/1     Running   0          70m
-mayastor-etcd-1   1/1     Running   0          70m
-mayastor-etcd-2   1/1     Running   0          70m
-```
-{% endtab %}
-{% endtabs %}
-
-## Deploy Mayastor Components
-
-### CSI Node Plugin
-
-{% tabs %}
-{% tab title="Command \(GitHub Latest\)" %}
-```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor/v1.0.2/deploy/csi-daemonset.yaml
-```
-{% endtab %}
-{% endtabs %}
-
-Verify that the CSI Node Plugin DaemonSet has been correctly deployed to all worker nodes in the cluster.
-
-{% tabs %}
-{% tab title="Command" %}
-```text
-kubectl -n mayastor get daemonset mayastor-csi
-```
-{% endtab %}
-
-{% tab title="Example Output" %}
-```text
-NAME           DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR              AGE
-mayastor-csi   3         3         3       3            3           kubernetes.io/arch=amd64   26m
-```
-{% endtab %}
-{% endtabs %}
-
-### Control Plane
-
-### Core Agents
-
-{% tabs %}
-{% tab title="Command \(GitHub Latest\)" %}
-```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/v1.0.2/deploy/core-agents-deployment.yaml
-```
-{% endtab %}
-{% endtabs %}
-
-Verify that the core agent pod is running.
-
-{% tabs %}
-{% tab title="Command" %}
-```text
-kubectl get pods -n mayastor --selector=app=core-agents
-```
-{% endtab %}
-
-{% tab title="Example Output" %}
-```text
-NAME                           READY   STATUS    RESTARTS   AGE
-core-agents-5f4d9f786b-6vvxc   1/1     Running   0          117s
-```
-{% endtab %}
-{% endtabs %}
-
-### REST
-
-{% tabs %}
-{% tab title="Command \(GitHub Latest\)" %}
-```text
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/v1.0.2/deploy/rest-deployment.yaml
-kubectl apply -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/v1.0.2/deploy/rest-service.yaml
-```
-{% endtab %}
-
-{% tab title="Example Output" %}
-```text
-deployment.apps/rest created
-service/rest created
-```
-{% endtab %}
-{% endtabs %}
-
-Verify that the REST pod is running.
+## Installation via helm
 
+1. Add the `mayastor-extensions` helm repository.
```

Comment:
```suggestion
1. Add the OpenEBS Mayastor Helm repository.
```
I think we shouldn't name the repository, since we may well change this in the near future.  The repo add command of course, can't be changed but its one less detail for the future.  Also, "Helm" is capitalised.